### ISSUE TITLE
Posting a journal does not set the 'posted' flag on the journal.

### ISSUE THREAD

**Issue by hansbak** [moqui/PopCommerce#47]
As the title says, entering a journal and click the 'post' button in the vapps/PopcAdmin/Accounting/FindJournal screen for the related journal posts fine, but then does not set the posted flag on the same line.

**Comment by jonesde** [comment]
I'm lost... maybe it would help to break it down in terms of:

1. Steps to Reproduce
2. What Happened
3. What Should Have Happened

As described here: https://moqui.org/m/docs/moqui/Issue+and+Pull+Request+Guide#bugs-and-general-issues

**Comment by hansbak** [comment]
Steps to reproduce:
1. create a journal. (http://localhost:8080/vapps/PopcAdmin/Accounting/FindJournal push 'New Journal)
2. post the created journal (push the 'post' button)

What happened: 
On the journal the Posted flag still 'N' and posted date still blank

What should have happened:
Posted flag should be 'Y' and posted date filled.

**Comment by jonesde** [comment]
That is correct, posting a journal posts all transactions in the journal, and by default does not 'close' the journal as this is not a common use case to date. If you are saying it is a use case important to you then the FindJournal screen could be changed to set the closeJournal parameter to true when calling the post#GlJournal service. That could be done either hard coded in a custom fork of SimpleScreens (which is where this screen lives in code, not in the PopCommerce repo), or for a more general change that could go back into the SimpleScreens repo add a dialog that asks the user if they want to close the journal or not.

**Comment by hansbak** [comment]
Thanks David.
